### DISCUSSION TITLE
how to make a vscode-like side bar?

### DISCUSSION THREAD

**Discussion by unixniu** [quasarframework/quasar#17339] [General - Components / Directives / etc]
- it's like layout drawer but belongs to the page, not on app level
- resizable by dragging
- collapsible to mini mode

nesting a layout with drawer inside a top-level layout doesn't seem to work.
any suggestion is appreciated.

**Comment by ivanjaros** [comment]
Maybe https://quasar.dev/vue-components/splitter with a bit of tweaking for mini mode.
